tmt/steps/provision/connect.py

```diff
@@ -79,6 +85,7 @@ def go(self) -> None:
             role=self.get('role'),
             guest=self.get('guest'),
             user=self.get('user'),
+            become=self.get('become'),
```

Comment:
Note: this will not be needed once https://github.com/teemtee/tmt/pull/2353 lands

Reply:
I saw the PR was merged. I'll update mine.